Gaijinizer patch/Map301.txt

```diff
@@ -287,7 +281,8 @@ even if it passes the near place considerably.#GT
 
 \\C[0]近くを通り過ぎる際は、意識してね。
 > CONTEXT: Map301/events/17/pages/0/27/Dialogue 
-Let's see...... As a rule of thumb...... \\C[2] 
-"surrounding two squares" is a range to be noticed.
-\\C[0] Be conscious when you pass near.#GT
+Let's see... As a rule of thumb... \\C[2]
+They have a "2 square radius" where they'll
+notice you.
+\\C[0]If you get closer than that, be careful.#Amaboye
```

Comment:
Just that last part might need a slight change.

"Let's see... As a rule of thumb... \\C[2]
They have a "2 square radius" where they'll
notice you. \\C[0]Be conscious of that when you're
passing close by them."